test/common/fit_common.py

```diff
@@ -559,17 +554,17 @@ def get_auth_token():
     api_login = {"username": fitcreds()["api"][0]["admin_user"], "password": fitcreds()["api"][0]["admin_pass"]}
     redfish_login = {"UserName": fitcreds()["api"][0]["admin_user"], "Password": fitcreds()["api"][0]["admin_pass"]}
     try:
-        restful("https://" + fitargs()['rackhd_host'] + ":" + str(API_PORT) +
+        restful(API_PROTOCOL + "://" + fitargs()['rackhd_host'] + ":" + str(API_PORT) +
                 "/login", rest_action="post", rest_payload=api_login, rest_timeout=2)
     except:
         AUTH_TOKEN = "Unavailable"
         return False
     else:
-        api_data = restful("https://" + fitargs()['rackhd_host'] + ":" + str(API_PORT) +
+        api_data = restful(API_PROTOCOL + "://" + fitargs()['rackhd_host'] + ":" + str(API_PORT) +
```

Comment:
I had another PR "https://github.com/RackHD/RackHD/pull/626" to set get auth token by https only. If 8080 http auth is note enabled in /opt/monorail/config.json , it will failed to get auth token by http.